README.md

```diff
@@ -24,12 +24,13 @@
 # Features
 
 1. Easy to use.
-2. Automatic formats detection.
-3. Same usage syntax for all formats.
-4. Uses encoding and decoding streams to improve performance.
-5. No runtime dependencies (for _Linux x86_64_).
-6. Can list archive contents with pretty tree formatting.
-7. Shell completions (soon!).
+2. Accessibility (A11Y) mode via `--accessibility` or env var `ACCESSIBILITY`
```

Comment:
So in the "Features" list we just note that there is an accessibility mode and link a new wiki page for that?

I like the idea of a separate wiki page.

I've been reading through a [case study: "Accessibility of Command Line Interfaces "](https://dl.acm.org/doi/fullHtml/10.1145/3411764.3445544) and one of their recommendations was to provide a detailed description of the tool output since visually impaired people can't parse the structure of the output easily. Such a thing could be described over there as well.

Reply:
Something like
```
2. Accessibility mode (A11Y) via `--accessibility` or `ACCESSIBILITY` env var (see [wiki page](https://github.com/ouch-org/ouch/wiki/Accessibility)).
```

2. Accessibility mode (A11Y) via `--accessibility` or `ACCESSIBILITY` env var (see [wiki page](https://github.com/ouch-org/ouch/wiki/Accessibility)).